packages/desktop/components/popups/VerifyLedgerTransactionPopup.svelte

```diff
@@ -5,9 +5,18 @@
     import { onDestroy } from 'svelte'
     import { showInternalVerificationPopup, resetShowInternalVerificationPopup } from '@core/ledger'
 
+    export let isEvmTransaction: boolean
+    export let useBlindSigning: boolean
+
+    // Regular transaction
     export let toAddress: string
     export let toAmount: string
+    export let chainId: string
+    export let maxFees: string
+
+    // Blindly signed transaction
     export let hash: string
+    export let bipPath: string
 
     const hasSendConfirmationProps = (toAddress && toAmount) || hash
```

Comment:
can one of these be updated while the component is loaded?
I guess its safer to just 
```suggestion
    $: hasSendConfirmationProps = (toAddress && toAmount) || hash
```

Reply:
They shouldn't be updated, since it's a popup and it receives all its arguments on opening it. I would argue against this change.